pkg/jws/common.go

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
I wonder if we should make this an internal package.

Reply:
I will look into making it internal in my next PR. I might need to expose some functionality.